socketshark/subscription.py

```diff
@@ -385,6 +414,14 @@ async def subscribe(self, event):
                 self.periodic_authorizer()
             )
 
+        if (
+            'heartbeat_period' in self.service_config
+            and 'on_heartbeat' in self.service_config
+        ):
+            self._periodic_heartbeat_task = asyncio.ensure_future(
```

Comment:
We have to cancel it in `cleanup_subscription()`.

Reply:
done